_includes/banner.html

```diff
@@ -1,60 +1,42 @@
 <section class="usa-banner" aria-label="Official government website">
   <div class="usa-accordion">
-    <div class="usa-banner__header">
+    <header class="usa-banner__header">
```

Comment:
Only discovered after rebasing the merged changes into my end-to-end testing branch at #469: This is a bit of a problem in that the banner partial is nested into a `<header>` element:

https://github.com/18F/identity-site/blob/42f73f91352d230db072d128835bcff611975b70/_layouts/main.html#L9-L15

Nesting `<header>` is not valid HTML ([see validator result](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdemo.login.gov%2Fwho-uses-login%2F)).

It's also an interesting challenge to resolve this while respecting a few other guidelines around landmarks:

- https://dequeuniversity.com/rules/axe/4.1/region
- https://dequeuniversity.com/rules/axe/4.1/landmark-no-duplicate-banner

I'm still poking at solutions, but at the moment I'm inclined to suggest we keep the main layout `<header>` wrapping this banner and the rest of the header (navigation, etc), then revert this change back to a `<div>`.

Reply:
Secondarily, my suggested change at https://github.com/18F/identity-site/pull/494#discussion_r547939335 was incorrect in that the `</header>` closing tag should have occurred sooner, to avoid including `usa-banner__content`.